src/model/gateway.rs

```diff
@@ -567,10 +547,8 @@ impl Serialize for Presence {
         }
 
         let mut state = serializer.serialize_struct("Presence", 5)?;
```

Comment:
The number should be updated to mirror the number of fields that are going to be serialized.
```suggestion
        let mut state = serializer.serialize_struct("Presence", 3)?;
```

Reply:
Done